src/Shared/FxPolyfills/FxPolyfills.targets

```diff
@@ -0,0 +1,29 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. -->
+<!-- The .NET Foundation licenses this file to you under the MIT license. -->
+
+<Project>
+  <PropertyGroup>
+    <IsTargetingNetFramework>false</IsTargetingNetFramework>
+    <IsTargetingNetFramework Condition="$(TargetFramework.StartsWith('net4'))">true</IsTargetingNetFramework>
+    <SharedFxPolyfillsDir>$(MSBuildThisFileDirectory)</SharedFxPolyfillsDir>
+  </PropertyGroup>
+
+  <!-- Include all Framework Extension files from the shared location, excluding test-specific ones -->
+  <ItemGroup Condition="$(IsTargetingNetFramework)">
+    <Reference Include="System.Net.Http" />
+
+    <Using Include="System.Net.Http" />
+    <Using Include="System.Threading.Tasks" />
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
These should only be included if targeting netfx no?

Reply:
This is so the files show up in VS. Each of the polyfills have a `#if NETFRAMEWORK` to exclude them there. I'd prefer to just not include the files, but it is helpful for them to show up in the solution explorer

Reply:
We should try to figure out how to get them to show without having to always have them in the solution explorer. Perhaps it would help pinging someone from MSBuild for help here?

Reply:
I went ahead and just included the targets conditionally. doesn't show up in the solution explorer, but probably not a big deal for the most part